src/app/(main)/mypage/page.tsx

```diff
@@ -1,40 +1,42 @@
 'use client'
 
+import { useQuery } from '@tanstack/react-query'
+
+import { userQueries } from '@/service/api/mypage'
+import { useMyInfoStore } from '@/store/myInfo'
+
 import { UserInfoSection, UserSocialInfoSection } from './_components'
 
 const MyPage = () => {
+  const { userId } = useMyInfoStore((state) => state.getMyInfo())
+
+  const { data: userInfo } = useQuery(userQueries.userInfo({ userId: userId }))
+
+  const { data: userProfile } = useQuery(
+    userQueries.profile({ userId: userId }),
+  )
```

Comment:
```suggestion
  
  const {
    data: userInfo,
    status: infoStatus,
    error: infoError,
  } = useQuery(userQueries.userInfo({ userId: userId }))

  const {
    data: userProfile,
    status: profileStatus,
    error: profileError,
  } = useQuery(userQueries.profile({ userId: userId }))

  if (infoStatus === 'pending' || profileStatus === 'pending') {
    return <Spinner />
  }

  if (infoError || profileError) {
    throw new Error(API_ERROR_MESSAGES.UNKNOWN_ERROR)
  }
```

query를 가져올 때 loading, error 처리를 추가해주세용